setup.php

```diff
@@ -588,9 +605,9 @@ function cloneClicked() {
 echo "<p>Welcome to LibreHealth EHR.  This utility will step you through the installation and configuration of LibreHealth EHR for your practice.</p>\n";
 echo "<ul><li>Before proceeding, be sure that you have a properly installed and configured MySQL server available, and a PHP configured webserver.</li>\n";
 
-echo "<li>Detailed installation instructions can be found in the <a href='INSTALL' target='_blank'><span STYLE='text-decoration: underline;'>'INSTALL'</span></a> manual file.</li>\n";
+echo "<li>Detailed installation instructions can be found in the <a href='Documentation/1_Installing/InstallingLibreEHR.html' target='_blank'><span STYLE='text-decoration: underline;'>'Installation'</span></a> manual file.</li>\n";
```

Comment:
We could just do includes or iframes in hidden divs with all the expanded information in it right there instead of giving links that cause tab-hogging.
`
    <a href="javascript:toggleDiv('main');" style="background-color: #ccc; padding: 5px 10px;">Show Me More</a>

    <div id="main" style="background-color: #F0F0F0; display: none; padding: 5px 10px;"><br>
<?php
include_once "whatever url";
?>
</div>`

Reply:
I guess that's a matter of opinion, I MUCH prefer to have a new Tab for docs not a split frame.